docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb

```diff
@@ -0,0 +1,206 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-12-03T17:20:12.074558Z",
+     "start_time": "2020-12-03T17:20:12.046560Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "db_file_path = os.path.join(os.getcwd(), 'plottr_for_live_plotting_tutorial.db')\n",
+    "initialise_or_create_database_at(db_file_path)\n",
+    "exp = load_or_create_experiment(experiment_name='plottr_for_live_plotting_with_subsecond_refresh_rate',\n",
+    "                          sample_name=\"no sample\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now open an anaconda prompt and activate your conda environment where plottr is installed. Start Inspectr GUI with `plottr-inspectr` command. You will see a window as show below. In this window, set `Refresh interval (s)` to the desired value and check `Auto-plot new`. Using `File` button load the database initialized (or created) above.\n",
+    "\n",
+    "<div>\n",
+    "<img src=\"files/plottr.jpg\" width=\"400\" height=\"400\"/>\n",
+    "</div>\n",
+    "\n",
+    "Start the measurement as shown below. Notice `meas.write_period = 0.1` . This sets the measurement write period to 0.1s and is recommended to be used with sub-second refresh interval for plottr.\n",
+    "\n",
+    "<div>\n",
+    "<img src=\"files/plottr_set_refresh_interval_and_autoplot.jpg\"/>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-12-03T16:00:53.947203Z",
+     "start_time": "2020-12-03T16:00:53.883203Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Starting experimental run with id: 1. \n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "meas = Measurement(exp=exp)\n",
+    "meas.register_parameter(dac.ch1)\n",
+    "meas.register_parameter(dmm.v1, setpoints=(dac.ch1,))\n",
+    "\n",
+    "meas.write_period = 0.1\n",
+    "\n",
+    "\n",
+    "with meas.run() as datasaver:\n",
+    "\n",
+    "    for set_v in np.linspace(0, 25, 10):\n",
+    "        dac.ch1.set(set_v)\n",
+    "        get_v = dmm.v1.get()\n",
+    "        datasaver.add_result((dac.ch1, set_v),\n",
+    "                             (dmm.v1, get_v))\n",
+    "        \n",
+    "    dataset = datasaver.dataset\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "On starting this measurement, a plot window (as shown below) will open automatically. This plot will keep refreshing at the interval rate set by you till the measurement runs.\n",
```

Comment:
and i think this nice exampke notebook can be concluded with somthling like "refer to plottr docs and whatever for more information on it's features"